src/Components/TodoItems.tsx

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Rename file to `TodoItem` 
<img width="125" alt="image" src="https://github.com/user-attachments/assets/fc239c88-2555-45c8-b431-f591d538334f" />